src/utils/plotSpecs/generateFeaturesVsUMIsScatterplot.js

```diff
@@ -132,14 +136,14 @@ const generateSpec = (config, plotData) => {
       },
     ],
     title:
-  {
-    text: config.title.text,
-    color: config.colour.masterColour,
-    anchor: config.title.anchor,
-    font: config.fontStyle.font,
-    dx: config.title.dx,
-    fontSize: config.title.fontSize,
-  },
+    {
+      text: config.title.text,
+      color: config.colour.masterColour,
+      anchor: config.title.anchor,
+      font: config.fontStyle.font,
+      dx: config.title.dx,
+      fontSize: config.title.fontSize,
+    },
```

Comment:
you sure it has to be nested 1 more level?

Reply:
Not sure. This is automated formatting (opened ui with `code ui.code-workspace`). If I try to tab these lines back by one they re-indent on save. Maybe my vscode is missing something though? Let me know if yours produces a different indentation.

Reply:
hmm, mine shows the same as yours. Maybe it is ok then